lib/timers.js

```diff
@@ -305,6 +306,9 @@ Timeout.prototype.unref = function() {
   if (this._handle) {
     this._handle.unref();
   } else if (typeof(this._onTimeout) === 'function') {
+    // Prevent running callback multiple times
+    // when unref() is called during the callback
+    if (this._called) return;
```

Comment:
Yes, I was unsure here too, but since I only wanted to prevent the `_onTimeout` call, these seemed right.